cmd/lambda/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"github.com/aws/aws-lambda-go/lambda"
+
+	sync "github.com/silinternational/personnel-sync/v6"
+)
+
+type LambdaConfig struct {
+	ConfigPath string
```

Comment:
Mind is blanking on how this gets set, but I see it's taken from the lambda example for sync, so giving approval anyway.

Reply:
I think that you can pass anything you want into the handler and it will unmarshal into your custom struct when the Lambda is invoked.